e2e/explorer/workflow/revisions/revisionsList.spec.ts

```diff
@@ -0,0 +1,241 @@
+});
+
+const actionNavigateToRevisions = async (page: Page) => {
+  await page.goto(`${namespace}/explorer/workflow/revisions/${workflow}`);
+};
+
+const actionNavigateToWorkflowEditor = async (page: Page) => {
+  await page.goto(`${namespace}/explorer/workflow/active/${workflow}`);
+};
+
+const actionCreateRevisionAndTag = async (page: Page) => {
+  await actionNavigateToWorkflowEditor(page);
+  await actionEditAndSaveWorkflow(page);
+  await actionMakeRevision(page);
+  await actionWaitForSuccessToast(page);
+  await actionNavigateToRevisions(page);
+
+  // open context menu & click on create-tag
+  const firstRevision = await page
+    .getByTestId(/workflow-revisions-link-item/)
+    .nth(1)
+    .innerText();
+  const firstItemMenuTrg = page.getByTestId(
+    `workflow-revisions-item-menu-trg-${firstRevision}`
+  );
+  await firstItemMenuTrg.click();
+
+  const createTagTrg = page.getByTestId(
+    `workflow-revisions-trg-create-tag-dlg-${firstRevision}`
+  );
+  await createTagTrg.click();
+
+  // type name and save & wait for the success toast
+  const inputName = page.getByTestId("dialog-create-tag-input-name");
+  const newTag = faker.random.alphaNumeric(9);
+  await inputName.type(newTag);
+  await page.getByTestId("dialog-create-tag-btn-submit").click();
+  await actionWaitForSuccessToast(page);
+  return [firstRevision, newTag];
+};
+
+test("it is possible to navigate to the revisions tab", async ({ page }) => {
+  await page.goto("/");
+  await expect(
+    page.getByTestId("breadcrumb-namespace"),
+    "it renders the breadcrumb for a namespace"
+  ).toBeVisible();
+  // at this point, any namespace may be loaded.
+  // let's navigate to the test's namespace via breadcrumbs.
+  await page.getByTestId("dropdown-trg-namespace").click();
+  await page
+    .getByRole("option", {
+      name: namespace,
+    })
+    .click();
+  await expect(page, "the namespace is reflected in the url").toHaveURL(
+    `/${namespace}/explorer/tree`
+  );
+  await expect(
+    page.getByTestId("breadcrumb-namespace"),
+    "the namespace is reflected in the breadcrumbs"
+  ).toHaveText(namespace);
+
+  await page.getByTestId(`explorer-item-link-${workflow}`).click();
+  const revisionsTab = page.getByTestId("workflow-tabs-trg-revisions");
+  await expect(
+    revisionsTab,
+    "screen should have activeRevision tab"
+  ).toBeVisible();
+  await revisionsTab.click();
+  await expect(page, "the workflow is reflected in the url").toHaveURL(
+    `${namespace}/explorer/workflow/revisions/${workflow}`
+  );
+});
+
+test("latest is the only revision by default", async ({ page }) => {
+  await actionNavigateToRevisions(page);
+  const revisions = page.getByTestId(/workflow-revisions-link-item-/);
+  await expect(revisions, "revisions should have the name latest").toHaveText(
+    "latest"
+  );
+  await expect(revisions, "number of revisions should be one").toHaveCount(1);
+});
+
+test("it is possible to revert to the previous the workflow", async ({
```

Comment:
This test is testing the wrong thing. It is part of the revisions list, but your test uses the revert button from the editor page.

Reply:
Yes, you are right
I have updated it

Reply:
The test did not pass on my machine, and also not in [ci](https://github.com/direktiv/direktiv-ui/actions/runs/5124962165/jobs/9217499775?pr=377). Did it work for you?

Reply:
Another small thing that goes into the category "you could not know that" is the following: the test for the revisions list requires us to have multiple revisions in place to just as a setup for our test. You solved that be reusing the code from the editor tests, that created multiple revisions from the users' perspective. That was the right approach when you started with the test, but we now have a solution that can make our test cleaner and independent of other UI parts. The goal is basically to use the api to create the test setup. Since you are not familiar with the api yet, you can ask me if you need to solve a specific test setup in the future but for the revisions list test, [I prepared an example for you](https://github.com/direktiv/direktiv-ui/pull/377/commits/80240db70fd5e90af5635ea688f6a168d4822246) that I would like you to apply on the other tests in that file too. The basic gist of my changes is the following

- I used `createWorkflowWithThreeRevisions` to create a workflow with three revisions (you can use the same method but you can also just inspect the function to get an idea how to use the api, if you know which endpoint to call, it's pretty straight forward)
- I also created a random workflow for the test with a random name in the very first line
- ~I removed the line "after click menu trigger, menu content should appear" because you click the delete button in the next step anyway and this only works when the menu is visible~ I changed my mind and added that test back into the code.  🙃

Reply:
Yes, it saves the test time